src/modules/resource_monitor.py

```diff
@@ -27,15 +30,20 @@ def _get_delay_query(self):
 
     def _get_delay(self):
         query = self._get_delay_query()
-        delay = self.psql_conn.select_single(query) or 0
-        return delay
+        value = self.loop.run_until_complete(
```

Comment:
Does syntax **asyncio.run** work in this case? So that we won't need self.loop in the class.

Reply:
No **Asyncio.run** don't work in this case. It will pop up error:
**cannot perform operation: another operation is in progress**